### PR TITLE
Simsopt Simulator class

### PR DESCRIPTION
**Pull request by joaobernardo23**

### CODE REVIEW

File: inductiva/stellarators/simulators/simsopt.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+"""Simsopt module of the API."""
+from typing import Optional
+from uuid import UUID
+
+from inductiva import types, tasks
+from inductiva.simulation import Simulator
+
+
+class Simsopt(Simulator):
+    """Class to invoke a generic Simsopt simulation on the API."""
```
Comment (fabiocruz) [inline inductiva/stellarators/simulators/simsopt.py:10]: Simplify, "Invokes a simsopt simulation on the API."

File: inductiva/stellarators/simulators/simsopt.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
+"""Simsopt module of the API."""
+from typing import Optional
+from uuid import UUID
+
+from inductiva import types, tasks
+from inductiva.simulation import Simulator
+
+
+class Simsopt(Simulator):
```
Comment (fabiocruz) [inline inductiva/stellarators/simulators/simsopt.py:9]: I think we should adapt this name in the future to reflect the specific use-case we're using simsopt for but for now this should be fine.

**Review by fabiocruz — APPROVED** [review]
LGTM!

File: inductiva/stellarators/simulators/simsopt.py (hunk 2 above)
Comment (joaobernardo23) [reply inductiva/stellarators/simulators/simsopt.py:9]: Noted!

**Pull request merged**